File: easy/023. The irritating Students/students.cpp
#include <bits/stdc++.h>
using namespace std;

int main()
{
	int total_cases = 0;
	cin >> total_cases;
	vector<int> students(total_cases, 0);

	for (int i = 0; i < total_cases; i++) {
		cin >> students[i];
	}

	for (auto student : students) {
		cout << student/2 << " " << student/2 + student%2 << endl;
	}
}

File: easy/023. The irritating Students/Students.java
import java.util.Scanner;

class Students {
	public static void main(String[] args) {
		Scanner input = new Scanner(System.in);
		int total_cases = input.nextInt();
		int[] students = new int[total_cases];

		for (int i = 0; i < total_cases; i++) {
			input.skip("\n");
			students[i] = input.nextInt();
		}

		for (int i = 0; i < total_cases; i++) {
			System.out.println(students[i]/2 + " " + (students[i]/2 + students[i]%2));
		}
	}
}